inmap/cmd/config.go

```diff
@@ -119,14 +121,24 @@ func ReadConfigFile(filename string) (config *ConfigData, err error) {
 	}
 
 	for k, v := range config.OutputVariables {
+		v = strings.Replace(v, "\r\n", " ", -1)
+		v = strings.Replace(v, "\n", " ", -1)
 		config.OutputVariables[os.ExpandEnv(k)] = os.ExpandEnv(v)
 	}
 
+	if config.VarGrid.PopConcThresholdEnv != "" {
```

Comment:
I'm not a huge fan of having a separate exported field for the environment variable. To me, it seems like the extra confusion for most users of trying to figure out what that field is for would outweigh the convenience of the few users who will want to experiment with different thresholds. I would say that for now we delete this, but create an issue to allow overriding configuration file variables using command line flags, perhaps using https://github.com/spf13/viper.